capplets/about-me/mate-about-me-password.c

```diff
@@ -238,11 +234,15 @@ spawn_passwd (PasswordDialog *pdialog, GError **error)
 	/* Turn off buffering */
 	g_io_channel_set_buffered (pdialog->backend_stdin, FALSE);
 	g_io_channel_set_buffered (pdialog->backend_stdout, FALSE);
+	g_io_channel_set_buffered (pdialog->backend_stderr, FALSE);
 
 	/* Add IO Channel watcher */
 	pdialog->backend_stdout_watch_id = g_io_add_watch (pdialog->backend_stdout,
 													   G_IO_IN | G_IO_PRI,
 													   (GIOFunc) io_watch_stdout, pdialog);
+	pdialog->backend_stderr_watch_id = g_io_add_watch (pdialog->backend_stderr,
+													   G_IO_IN | G_IO_PRI,
```

Comment:
Can you please fix indents when you already touch this function?
I know indent issue was there before.....